smalltalksrc/VMMaker/DruidJIT.class.st

```diff
@@ -26,46 +26,262 @@ DruidJIT class >> bytecodeTable [
-		  { 1. 47. 47. #gen_PushLiteralConstantBytecode15. #needsFrameNever:. 1 }.
-		  { 1. 48. 48. #gen_PushLiteralConstantBytecode16. #needsFrameNever:. 1 }.
-		  { 1. 49. 49. #gen_PushLiteralConstantBytecode17. #needsFrameNever:. 1 }.
-		  { 1. 50. 50. #gen_PushLiteralConstantBytecode18. #needsFrameNever:. 1 }.
-		  { 1. 51. 51. #gen_PushLiteralConstantBytecode19. #needsFrameNever:. 1 }.
-		  { 1. 52. 52. #gen_PushLiteralConstantBytecode20. #needsFrameNever:. 1 }.
-		  { 1. 53. 53. #gen_PushLiteralConstantBytecode21. #needsFrameNever:. 1 }.
-		  { 1. 54. 54. #gen_PushLiteralConstantBytecode22. #needsFrameNever:. 1 }.
-		  { 1. 55. 55. #gen_PushLiteralConstantBytecode23. #needsFrameNever:. 1 }.
-		  { 1. 56. 56. #gen_PushLiteralConstantBytecode24. #needsFrameNever:. 1 }.
-		  { 1. 57. 57. #gen_PushLiteralConstantBytecode25. #needsFrameNever:. 1 }.
-		  { 1. 58. 58. #gen_PushLiteralConstantBytecode26. #needsFrameNever:. 1 }.
-		  { 1. 59. 59. #gen_PushLiteralConstantBytecode27. #needsFrameNever:. 1 }.
-		  { 1. 60. 60. #gen_PushLiteralConstantBytecode28. #needsFrameNever:. 1 }.
-		  { 1. 61. 61. #gen_PushLiteralConstantBytecode29. #needsFrameNever:. 1 }.
-		  { 1. 62. 62. #gen_PushLiteralConstantBytecode30. #needsFrameNever:. 1 }.
-		  { 1. 63. 63. #gen_PushLiteralConstantBytecode31. #needsFrameNever:. 1 }.
-		  { 1. 64. 75. #unknownBytecode }.
-		  { 1. 76. 76. #gen_PushReceiverBytecode. #needsFrameNever:. 1 }.
-		  { 1. 77. 91. #unknownBytecode }.
-		  { 1. 92. 92. #gen_ReturnTopFromMethod. #return. #isMappedInBlock. #needsFrameIfInBlock:. -1 }.
-		  { 1. 93. 223. #unknownBytecode }.
-		  { 2. 224. 247. #unknownBytecode }.
-		  { 3. 248. 255. #unknownBytecode } }
+		  { 1. 0. 0. #unknownBytecode }.
+		  { 1. 1. 1. #unknownBytecode }.
+		  { 1. 2. 2. #unknownBytecode }.
+		  { 1. 3. 3. #unknownBytecode }.
+		  { 1. 4. 4. #unknownBytecode }.
+		  { 1. 5. 5. #unknownBytecode }.
+		  { 1. 6. 6. #unknownBytecode }.
+		  { 1. 7. 7. #unknownBytecode }.
+		  { 1. 8. 8. #unknownBytecode }.
+		  { 1. 9. 9. #unknownBytecode }.
+		  { 1. 10. 10. #unknownBytecode }.
+		  { 1. 11. 11. #unknownBytecode }.
+		  { 1. 12. 12. #unknownBytecode }.
+		  { 1. 13. 13. #unknownBytecode }.
+		  { 1. 14. 14. #unknownBytecode }.
+		  { 1. 15. 15. #unknownBytecode }.
+		  { 1. 16. 16. #gen_PushLiteralVariable16CasesBytecode0 }.
+		  { 1. 17. 17. #unknownBytecode }.
+		  { 1. 18. 18. #unknownBytecode }.
+		  { 1. 19. 19. #unknownBytecode }.
+		  { 1. 20. 20. #unknownBytecode }.
+		  { 1. 21. 21. #unknownBytecode }.
+		  { 1. 22. 22. #unknownBytecode }.
+		  { 1. 23. 23. #unknownBytecode }.
+		  { 1. 24. 24. #unknownBytecode }.
+		  { 1. 25. 25. #unknownBytecode }.
+		  { 1. 26. 26. #unknownBytecode }.
+		  { 1. 27. 27. #unknownBytecode }.
+		  { 1. 28. 28. #unknownBytecode }.
+		  { 1. 29. 29. #unknownBytecode }.
+		  { 1. 30. 30. #unknownBytecode }.
+		  { 1. 31. 31. #unknownBytecode }.
+		  { 1. 32. 32. #gen_PushLiteralConstantBytecode0 }.
```

Comment:
We can generate all of these. Or do you want to measure only the list?

Reply:
Oh yes, I go by the list :D

Reply:
I know we can, we are super great ;) but I'm a maniac